pkg/cmd/cli/describe/chaindescriber_test.go

```diff
@@ -30,6 +30,19 @@ func TestChainDescriber(t *testing.T) {
 		includeInputImg  bool
 	}{
 		{
+			testName:         "circular test",
+			namespaces:       sets.NewString("example"),
+			output:           "",
+			defaultNamespace: "example",
+			name:             "ruby-22-centos7",
+			tag:              "latest",
+			path:             "../../../api/graph/test/circular.yaml",
+			humanReadable: map[string]int{
+				"Cycle detected in build configurations: BuildConfig|example/ruby-22-centos7 -> ImageStreamTag|example/ruby-hello-world:latest -> BuildConfig|example/ruby-hello-world -> ImageStreamTag|example/ruby-something-else:latest -> BuildConfig|example/ruby-something-else -> ImageStreamTag|example/ruby-22-centos7:latest -> BuildConfig|example/ruby-22-centos7": 1,
```

Comment:
Interesting, the analysis is calling `ResourceName()`:  https://github.com/openshift/origin/blob/master/pkg/build/graph/analysis/bc.go#L148
That produces the `kind|namespace/name` format per https://github.com/openshift/origin/blob/master/pkg/api/graph/types.go#L32

Got to break for a bit, but I'll dig into what produces the `bc/ruby-22-centos7` format and see about switching to that.